src/Pages/Login/Login.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react'
+import './Login.scss'
+
+class Login extends Component {
+  constructor() {
+    super()
+    this.state = {
+      id: '',
+      pw: '',
+      idValidation: '',
+      pwValidation: '',
+      keepLogin: false,
+      onOffBtn: false,
+    }
+  }
+
+  handleIdChange = (event) => {
+    this.setState({
+      id: event.target.value,
+    })
+  }
+
+  handlePwChange = (event) => {
+    this.setState({
+      pw: event.target.value,
+    })
+  }
+
+  handleValidation = (event) => {
+    event.preventDefault()
+    const checkId = this.state.id
+    const checkPw = this.state.pw
+    this.setState({
+      idValidation: !checkId ? '아이디를 입력해주세요.' : '',
+    })
+    this.setState({
+      pwValidation: !checkPw ? '비밀번호를 입력해주세요.' : '',
+    })
+    if (checkId && checkPw) {
+      this.props.history.push('/')
+    }
+  }
+
+  handleKeepLogin = () => {
+    this.setState({
+      keepLogin: !this.state.keepLogin,
+    })
+  }
+
+  handleOnOffBtn = () => {
+    this.setState({
+      onOffBtn: !this.state.onOffBtn,
+    })
+  }
```

Comment:
하나의 함수로 합쳐서 조건에 따라서 (e.target.id) 각각에 맞는 setState 설정을 해줄 수 있을것 같습니다 
```suggestion
  changeLoginStatus = (e)=> {
    const {keepLogin, onOffBtn} = this.state;
    const targetStatus = e.target.id;
targetStatus === "keepLogin" ? this.setState({keepLogin:!keepLogin}) : this.setState({onOffBtn:!onOffBtn})
   
  }
  
  
  ....
  
    <i  id='keepLogin' /> 
              ...
<span id='onOffBtn'> 
              ...
```